go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cilium/cilium-cli
 
-go 1.21.3
+go 1.21.4
```

Comment:
This change will conflict. It looks like it got bumped transitively from Tetragon.